packages/store/tests/property-selectors.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { Injectable } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { MockState } from 'packages/hmr-plugin/src/tests/hmr-mock';
+import { NgxsModule, State, Store } from '..';
+import { createPickSelector } from '../src/selectors/pick-selector';
+import { createPropertySelectors } from '../src/selectors/property-selectors';
+
+describe.only('Pick Selector', () => {
+  interface MyStateModel {
+    property1: string;
+    property2: number[];
+  }
+
+  @State<MyStateModel>({
+    name: 'myState',
+    defaults: {
+      property1: 'testValue',
+      property2: [1, 2, 3]
+    }
+  })
+  @Injectable()
+  class MyState {}
+
+  it('Passing null', () => {
+    const slices = createPropertySelectors(null);
+
+    expect(slices).toBe(null);
+  });
+
+  it('Passing undefined', () => {
+    const slices = createPropertySelectors(undefined);
+
+    expect(slices).toBe(null);
+  });
+
+  it('Passing empty object', () => {
+    const slices = createPropertySelectors({});
```

Comment:
Passing through anything that could potentially be a selector (a function or an object) should result in a selector being returned.

I guess the other 'empty' test case here is if the selector that you are slicing returns an empty object, or an object that does not contain the property that you are slicing out. Then the slice selector for the missing property should return undefined.